packages/spec-test-util/src/downloadTests.ts

```diff
@@ -85,7 +85,7 @@ export async function downloadGenericSpecTests<TestNames extends string>(
           log(`Downloaded  ${url}`);
         },
         {
-          retries: 2,
+          retries: 3,
```

Comment:
This used [async-retry](https://www.npmjs.com/package/async-retry) library, keeping retries as is to not change previous behavior